cmd/ocsp-responder/main.go

```diff
@@ -50,6 +50,25 @@ type Config struct {
 		// upstream's timeout when making request to ocsp-responder.
 		Timeout cmd.ConfigDuration
 
+		// The worst-case freshness of a response during normal operations.
+		//
+		// This controls behavior when both Redis and MariaDB backends are
+		// configured. If a MariaDB response is older than this, ocsp-responder
+		// will try to serve a fresher response from Redis, waiting for a Redis
+		// response if necessary.
+		//
+		// This is related to OCSPMinTimeToExpiry in ocsp-updater's config,
```

Comment:
```suggestion
		// This is related to ExpectedFreshness in ocsp-updater's config,
```

Reply:
Kept as-is, since I reverted the rename of OCSPMinTimeToExpiry.